course-mw/course-actors-common/src/main/java/org/sunbird/learner/constants/CourseJsonKey.java

```diff
@@ -33,5 +33,7 @@ public abstract class CourseJsonKey {
       "sunbird_send_email_notifictaion_api";
   public static final String NOTIFY_TEMPLATE="notifyTemplate";
   public static final String CERT_TEMPLATES = "certTemplates";
-    public static final String ADDITIONAL_PROPS = "additionalProps";
+  public static final String ADDITIONAL_PROPS = "additionalProps";
+  public static final String BATCH_ATTRIBUTES = "batchAttributes";
+  public static final String BATCH_ATTRIBUTES_COLUMN = "batchattributes";
```

Comment:
just need to add mapping in cassandratablecolumn.properties file.... 
you can send column as capital one when writing into cassandra -- it accepts... 
while reading it will use the cassandratablecolumn.properties file and it will return camel case key instead of lower case